docs/source/contribution.rst

```diff
@@ -171,6 +171,9 @@ In order to run unit tests at the repository root, you first have to build Cytho
 
   See `ccache <https://ccache.samba.org/>`_ for details.
 
+  If you want to use ccache for nvcc, please install ccache v3.3 or later.
+  You also need to set environment variable ``NVCC=/path/to/nvcc_with_ccache``.
```

Comment:
What is `nvcc_with_ccache`? In my environment `NVCC='ccache nvcc' python setup.py develop` works.

Reply:
`NVCC` argument only accepts single executable file path.

Reply:
`NVCC` only accepts single executable file path.
If we set `NVCC='ccache nvcc'`, `os.path.exists(nvcc)` become `False`.

Reply:
We need to support such types of values. In `distutils.ccompiler.CCompiler.set_executable`, environment variables are split with `distutils.util.split_quoted`.

Reply:
Thank you for your information. I fixed.